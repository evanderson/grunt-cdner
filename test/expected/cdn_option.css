h1 {
  background-image: url(//my.cdn.example.com/hello.png);
}
