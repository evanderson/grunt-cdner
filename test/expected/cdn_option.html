<!doctype html>
<html>
  <head>
    <title>test</title>
    <script src="//my.cdn.example.com/tmp/test.js"></script>
    <script>
      console.log('testing');
    </script>
    <link href="//my.cdn.example.com/tmp/test.css" rel="stylesheet">
    <link href="//cdn.1.test.com/test.css" rel="stylesheet">
  </head>
  <body>
    <img src="//my.cdn.example.com/tmp/testing.png" alt="">
    <img src="//my.cdn.example.com/tmp/testing.gif" alt="">
    <img src="//my.cdn.example.com/tmp/testing.jpg" alt="">
    <img src="//my.cdn.example.com/tmp/testing.jpeg" alt="">
    <script src="//my.cdn.example.com/tmp/test2.js"></script>
    <script src="//my.cdn.example.com/tmp/no/cdn/test3.js"></script>
    <script src="//cdn.2.test.com/test2.js"></script>
  </body>
</html>
